compose-builder/common-security.env

```diff
@@ -17,5 +17,6 @@
 #
 
 EDGEX_SECURITY_SECRET_STORE=true
+ENABLE_REGISTRY_ACL=true
```

Comment:
I thought we were going to leave this as FALSE until we knew we could get it all in in this release?

Reply:
the value to turn on of not is release time determination; right now we just want to go through the TAF test firsts.